src/sql_engine/src/lib.rs

```diff
@@ -151,8 +154,23 @@ impl<P: BackendStorage> Handler<P> {
                                 .collect()
                         })
                         .collect();
-                    let len = to_insert.len();
-                    match (self.storage.lock().unwrap()).insert_into(&schema_name, &name, to_insert)? {
+
+                    let specified_columns = if columns.is_empty() {
```

Comment:
This is more a question rather than I am asking for a changes. Have you considered to pass empty `Vec<String>` for column names?

Why I am asking is that I think `insert` and `update` probably, might be, possibly use similar logic.
For example:
`update_all` requires `Vec<(String, String)>` as one of a params, which is list of (`column name`, `column value`)
`insert_into` also could be changed to accept `Vec<(String, String)>` as list of (`column name`, `column value`)
by:
1) constructing it if user provide list of column names
2) reading column names in `sql_engine` as it's done for `select * from table` query and combining it with user provided values

WDYT?

Reply:
Yes. I think you are right. I will try to fix passing into a parameter of type `Vec<String, String>` like `update_all`. Thanks.

Reply:
Hi @alex-dukhno , I was working on fixing this, but it seems that `insert_into` could not unify with `update_all`. Since `insert_into` could insert multiple rows once. I thought if I could pass into the type of `Vec<Vec<String, String>>` (Multiple rows and each row including Column and Value). But it causes resolving column index for multiple times in `storage`.

So I try to fix it by passing into an empty column if no column specified. What's your opinion?

Reply:
Hi @alex-dukhno , I updated the signature of function `insert_into` as below, and passed into an empty Vec if no column specified.

Please fell free to add your comments. I could not confirm if it is the best solution. Thanks.

```
    pub fn insert_into(
        &mut self,
        schema_name: &str,
        table_name: &str,
        columns: Vec<String>,
        rows: Vec<Vec<String>>,
    ) -> SystemResult<Result<(), OperationOnTableError>> {
```